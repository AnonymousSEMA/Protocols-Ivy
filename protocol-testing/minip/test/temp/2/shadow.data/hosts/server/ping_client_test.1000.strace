00:00:00.000000000 [tid 1000] shadow_init_memory_manager(...) = 0
000000000000000000 [tid 1000] gettimeofday(...) = 0
00:00:00.000000000 [tid 1000] openat(-100, "/sys/devices/system/cpu", O_NONBLOCK | O_DIRECTORY | O_CLOEXEC, Mode(0x0)) = 3
00:00:00.000000000 [tid 1000] fstat(...) = 0
00:00:00.000000000 [tid 1000] getdents64(...) = 752
00:00:00.000000000 [tid 1000] getdents64(...) = 0
00:00:00.000000000 [tid 1000] close(3) = 0
00:00:00.000000000 [tid 1000] sched_getaffinity(258, 8, 0x5555555f1070) = -3 (ESRCH)
000000000000000000 [tid 1000] clock_gettime(...) = 0
00:00:00.000000000 [tid 1000] openat(-100, "/sys/devices/system/cpu/online", O_CLOEXEC, Mode(0x0)) = 3
00:00:00.000000000 [tid 1000] read(3, 0x7fffffffc4c0, 8192) = 2
00:00:00.000000000 [tid 1000] close(3) = 0
00:00:00.000000000 [tid 1000] futex(...) = 0
00:00:00.000000000 [tid 1000] futex(...) = 0
00:00:00.000000000 [tid 1000] setrlimit(...) = <native>
00:00:00.000000000 [tid 1000] ^^^ = SysCallReg { as_i64: 0, as_u64: 0, as_ptr: shadow_shim_helper_rs::syscall_types::ForeignPtr<()> { val: 0 } }
000000000000000000 [tid 1000] time(...) = 946684800
00:00:00.000000000 [tid 1000] rt_sigaction(...) = 0
00:00:00.000000000 [tid 1000] rt_sigaction(...) = 0
00:00:00.000000000 [tid 1000] write(2, "\"", 1) = 1
00:00:00.000000000 [tid 1000] write(2, "http_response_file.txt", 22) = 22
00:00:00.000000000 [tid 1000] write(2, "\"", 1) = 1
00:00:00.000000000 [tid 1000] write(2, "\n", 1) = 1
00:00:00.000000000 [tid 1000] write(2, "PROOTPATH 5b ", 13) = 13
00:00:00.000000000 [tid 1000] write(2, "/PFV", 4) = 4
00:00:00.000000000 [tid 1000] write(2, "\n", 1) = 1
00:00:00.000000000 [tid 1000] write(2, "\"", 1) = 1
00:00:00.000000000 [tid 1000] write(2, "/PFV/QUIC-Ivy-Attacker/doc/examples/quic"..., 68) = 68
00:00:00.000000000 [tid 1000] write(2, "\"", 1) = 1
00:00:00.000000000 [tid 1000] write(2, "\n", 1) = 1
00:00:00.000000000 [tid 1000] open("/PFV/QUIC-Ivy-Attacker/doc/examples/quic"..., O_RDWR, Mode(0x0)) = -2 (ENOENT)
00:00:00.000000000 [tid 1000] write(2, "cannot open file: No such file or direct"..., 44) = 44
00:00:00.000000000 [tid 1000] write(2, "-1", 2) = 2
00:00:00.000000000 [tid 1000] write(2, "\n", 1) = 1
00:00:00.000000000 [tid 1000] brk(0x55555562d000) = <native>
00:00:00.000000000 [tid 1000] ^^^ = SysCallReg { as_i64: 93824993120256, as_u64: 93824993120256, as_ptr: shadow_shim_helper_rs::syscall_types::ForeignPtr<()> { val: 93824993120256 } }
00:00:00.000000000 [tid 1000] brk(0x55555564e000) = <native>
00:00:00.000000000 [tid 1000] ^^^ = SysCallReg { as_i64: 93824993255424, as_u64: 93824993255424, as_ptr: shadow_shim_helper_rs::syscall_types::ForeignPtr<()> { val: 93824993255424 } }
00:00:00.000000000 [tid 1000] brk(0x555555670000) = <native>
00:00:00.000000000 [tid 1000] ^^^ = SysCallReg { as_i64: 93824993394688, as_u64: 93824993394688, as_ptr: shadow_shim_helper_rs::syscall_types::ForeignPtr<()> { val: 93824993394688 } }
00:00:00.000000000 [tid 1000] brk(0x555555691000) = <native>
00:00:00.000000000 [tid 1000] ^^^ = SysCallReg { as_i64: 93824993529856, as_u64: 93824993529856, as_ptr: shadow_shim_helper_rs::syscall_types::ForeignPtr<()> { val: 93824993529856 } }
00:00:00.000000000 [tid 1000] brk(0x5555556b2000) = <native>
00:00:00.000000000 [tid 1000] ^^^ = SysCallReg { as_i64: 93824993665024, as_u64: 93824993665024, as_ptr: shadow_shim_helper_rs::syscall_types::ForeignPtr<()> { val: 93824993665024 } }
00:00:00.000000000 [tid 1000] brk(0x5555556d3000) = <native>
00:00:00.000000000 [tid 1000] ^^^ = SysCallReg { as_i64: 93824993800192, as_u64: 93824993800192, as_ptr: shadow_shim_helper_rs::syscall_types::ForeignPtr<()> { val: 93824993800192 } }
00:00:00.000000000 [tid 1000] brk(0x5555556f4000) = <native>
00:00:00.000000000 [tid 1000] ^^^ = SysCallReg { as_i64: 93824993935360, as_u64: 93824993935360, as_ptr: shadow_shim_helper_rs::syscall_types::ForeignPtr<()> { val: 93824993935360 } }
00:00:00.000000000 [tid 1000] brk(0x555555715000) = <native>
00:00:00.000000000 [tid 1000] ^^^ = SysCallReg { as_i64: 93824994070528, as_u64: 93824994070528, as_ptr: shadow_shim_helper_rs::syscall_types::ForeignPtr<()> { val: 93824994070528 } }
00:00:00.000000000 [tid 1000] brk(0x555555736000) = <native>
00:00:00.000000000 [tid 1000] ^^^ = SysCallReg { as_i64: 93824994205696, as_u64: 93824994205696, as_ptr: shadow_shim_helper_rs::syscall_types::ForeignPtr<()> { val: 93824994205696 } }
00:00:00.000000000 [tid 1000] brk(0x555555757000) = <native>
00:00:00.000000000 [tid 1000] ^^^ = SysCallReg { as_i64: 93824994340864, as_u64: 93824994340864, as_ptr: shadow_shim_helper_rs::syscall_types::ForeignPtr<()> { val: 93824994340864 } }
000000000000000000 [tid 1000] clock_gettime(...) = 0
00:00:00.000000000 [tid 1000] brk(0x555555787000) = <native>
00:00:00.000000000 [tid 1000] ^^^ = SysCallReg { as_i64: 93824994537472, as_u64: 93824994537472, as_ptr: shadow_shim_helper_rs::syscall_types::ForeignPtr<()> { val: 93824994537472 } }
00:00:00.000000000 [tid 1000] brk(0x5555557aa000) = <native>
00:00:00.000000000 [tid 1000] ^^^ = SysCallReg { as_i64: 93824994680832, as_u64: 93824994680832, as_ptr: shadow_shim_helper_rs::syscall_types::ForeignPtr<()> { val: 93824994680832 } }
000000000000000000 [tid 1000] clock_gettime(...) = 0
00:00:00.000000000 [tid 1000] brk(0x5555557d4000) = <native>
00:00:00.000000000 [tid 1000] ^^^ = SysCallReg { as_i64: 93824994852864, as_u64: 93824994852864, as_ptr: shadow_shim_helper_rs::syscall_types::ForeignPtr<()> { val: 93824994852864 } }
00:00:00.000000000 [tid 1000] brk(0x5555557f5000) = <native>
00:00:00.000000000 [tid 1000] ^^^ = SysCallReg { as_i64: 93824994988032, as_u64: 93824994988032, as_ptr: shadow_shim_helper_rs::syscall_types::ForeignPtr<()> { val: 93824994988032 } }
000000000000000000 [tid 1000] clock_gettime(...) = 0
00:00:00.000000000 [tid 1000] brk(0x555555823000) = <native>
00:00:00.000000000 [tid 1000] ^^^ = SysCallReg { as_i64: 93824995176448, as_u64: 93824995176448, as_ptr: shadow_shim_helper_rs::syscall_types::ForeignPtr<()> { val: 93824995176448 } }
00:00:00.000000000 [tid 1000] brk(0x555555844000) = <native>
00:00:00.000000000 [tid 1000] ^^^ = SysCallReg { as_i64: 93824995311616, as_u64: 93824995311616, as_ptr: shadow_shim_helper_rs::syscall_types::ForeignPtr<()> { val: 93824995311616 } }
000000000000000000 [tid 1000] clock_gettime(...) = 0
00:00:00.000000000 [tid 1000] brk(0x555555865000) = <native>
00:00:00.000000000 [tid 1000] ^^^ = SysCallReg { as_i64: 93824995446784, as_u64: 93824995446784, as_ptr: shadow_shim_helper_rs::syscall_types::ForeignPtr<()> { val: 93824995446784 } }
00:00:00.000000000 [tid 1000] brk(0x555555886000) = <native>
00:00:00.000000000 [tid 1000] ^^^ = SysCallReg { as_i64: 93824995581952, as_u64: 93824995581952, as_ptr: shadow_shim_helper_rs::syscall_types::ForeignPtr<()> { val: 93824995581952 } }
00:00:00.000000000 [tid 1000] brk(0x5555558a7000) = <native>
00:00:00.000000000 [tid 1000] ^^^ = SysCallReg { as_i64: 93824995717120, as_u64: 93824995717120, as_ptr: shadow_shim_helper_rs::syscall_types::ForeignPtr<()> { val: 93824995717120 } }
000000000000000000 [tid 1000] clock_gettime(...) = 0
00:00:00.000000000 [tid 1000] brk(0x5555558c8000) = <native>
00:00:00.000000000 [tid 1000] ^^^ = SysCallReg { as_i64: 93824995852288, as_u64: 93824995852288, as_ptr: shadow_shim_helper_rs::syscall_types::ForeignPtr<()> { val: 93824995852288 } }
00:00:00.000000000 [tid 1000] brk(0x5555558fc000) = <native>
00:00:00.000000000 [tid 1000] ^^^ = SysCallReg { as_i64: 93824996065280, as_u64: 93824996065280, as_ptr: shadow_shim_helper_rs::syscall_types::ForeignPtr<()> { val: 93824996065280 } }
000000000000000000 [tid 1000] clock_gettime(...) = 0
00:00:00.000000000 [tid 1000] brk(0x555555930000) = <native>
00:00:00.000000000 [tid 1000] ^^^ = SysCallReg { as_i64: 93824996278272, as_u64: 93824996278272, as_ptr: shadow_shim_helper_rs::syscall_types::ForeignPtr<()> { val: 93824996278272 } }
00:00:00.000000000 [tid 1000] brk(0x555555951000) = <native>
00:00:00.000000000 [tid 1000] ^^^ = SysCallReg { as_i64: 93824996413440, as_u64: 93824996413440, as_ptr: shadow_shim_helper_rs::syscall_types::ForeignPtr<()> { val: 93824996413440 } }
000000000000000000 [tid 1000] clock_gettime(...) = 0
000000000000000000 [tid 1000] clock_gettime(...) = 0
000000000000000000 [tid 1000] clock_gettime(...) = 0
000000000000000000 [tid 1000] clock_gettime(...) = 0
000000000000000000 [tid 1000] clock_gettime(...) = 0
000000000000000000 [tid 1000] clock_gettime(...) = 0
000000000000000000 [tid 1000] clock_gettime(...) = 0
00:00:00.000000000 [tid 1000] socket(Inet, 2, 0) = 3
00:00:00.000000000 [tid 1000] write(2, "SOCKET ", 7) = 7
00:00:00.000000000 [tid 1000] write(2, "3", 1) = 1
00:00:00.000000000 [tid 1000] write(2, "\n", 1) = 1
00:00:00.000000000 [tid 1000] getsockname(3, 0x7fffffffd9a0, 16 (0x7fffffffd988)) = 0
00:00:00.000000000 [tid 1000] write(2, "source port number ", 19) = 19
00:00:00.000000000 [tid 1000] write(2, "0", 1) = 1
00:00:00.000000000 [tid 1000] write(2, "\n", 1) = 1
00:00:00.000000000 [tid 1000] write(2, "binding client id: ", 19) = 19
00:00:00.000000000 [tid 1000] write(2, "server", 6) = 6
00:00:00.000000000 [tid 1000] write(2, " addr: ", 7) = 7
00:00:00.000000000 [tid 1000] write(2, "184549377", 9) = 9
00:00:00.000000000 [tid 1000] write(2, " port: ", 7) = 7
00:00:00.000000000 [tid 1000] write(2, "4443", 4) = 4
00:00:00.000000000 [tid 1000] write(2, "\n", 1) = 1
00:00:00.000000000 [tid 1000] bind(3, 11.0.0.1:4443, 16) = 0
00:00:00.000000000 [tid 1000] getsockname(3, 0x7fffffffd9a0, 16 (0x7fffffffd988)) = 0
00:00:00.000000000 [tid 1000] write(2, "source port number ", 19) = 19
00:00:00.000000000 [tid 1000] write(2, "4443", 4) = 4
00:00:00.000000000 [tid 1000] write(2, "\n", 1) = 1
00:00:00.000000000 [tid 1000] write(2, "installing reader 2", 19) = 19
00:00:00.000000000 [tid 1000] write(2, "\n", 1) = 1
00:00:00.000000000 [tid 1000] write(2, "COUCOU", 6) = 6
00:00:00.000000000 [tid 1000] write(2, "\n", 1) = 1
00:00:00.000000000 [tid 1000] lseek(...) = -9 (EBADF)
00:00:00.000000000 [tid 1000] write(2, "cannot seek to beginning of file: Bad fi"..., 54) = 54
00:00:00.000000000 [tid 1000] read(-1, 0x5555555eba20, 1024) = -9 (EBADF)
00:00:00.000000000 [tid 1000] write(2, "recvfrom failed\n", 16) = 16
00:00:00.000000000 [tid 1000] exit_group(...) = <native>
