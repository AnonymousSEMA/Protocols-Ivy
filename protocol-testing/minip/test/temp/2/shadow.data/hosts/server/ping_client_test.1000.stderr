"http_response_file.txt"
PROOTPATH 5b /PFV
"/PFV/QUIC-Ivy-Attacker/doc/examples/quic/ping/http_response_file.txt"
cannot open file: No such file or directory
-1
SOCKET 3
source port number 0
binding client id: server addr: 184549377 port: 4443
source port number 4443
installing reader 2
COUCOU
cannot seek to beginning of file: Bad file descriptor
recvfrom failed
